app/models/ocelot/stanzas/CalloutStanza.scala

```diff
@@ -25,20 +25,24 @@ case class CalloutStanza(noteType: CalloutType, text: Int, override val next: Se
 
 object CalloutStanza {
 
-  implicit val calloutReads: Reads[CalloutStanza] = {
-
+  implicit val calloutReads: Reads[CalloutStanza] =
     ((JsPath \ "noteType").read[CalloutType] and
       (JsPath \ "text").read[Int] and
       (JsPath \ "next").read[Seq[String]](minLength[Seq[String]](1)) and
       (JsPath \ "stack").read[Boolean])(CalloutStanza.apply _)
 
-  }
+  implicit val owrite: OWrites[CalloutStanza] =
```

Comment:
Why are we using OWrites here and Writes elsewhere?

Reply:
OWrites are when you know the result is a JsObject, I was having trouble doing the Writes for Stanza and I recieved a compiler message suggesting I provide an OWites or OFormat method. The OWrites solved the problem